File: mar_long/groups.py
t = int(input())
for _ in range(t):
    s = input()
    groups = 0
    n = len(s)
    running = False
    for i in range(n):
        if s[i]=='1' and running == False:
            running = True
            groups+=1
        elif s[i]=='0':
            running = False
    print(groups)

File: mar_long/groups.cpp
#include <iostream>
using namespace std;

int main() {
    int t; cin>>t;
    while(t--) {
        string s; cin>>s;
        int n = s.length();
        int groups = 0;
        bool running = false; // counting of a grp going on or not
        for(int i=0; i<n; i++) {
            if(s[i]=='1' && running == false) {
                running = true;
                groups++;
            }

            else if(s[i]=='0') {
                running = false;
            }
        }
        
        cout<<groups<<'\n';
    }
}